metrics-operator/config/samples/metrics_v1alpha1_keptnmetric.yaml

```diff
@@ -7,3 +7,5 @@ spec:
     name: prometheus
   query: "sum(kube_pod_container_resource_limits{resource='cpu'}) - sum(kube_node_status_capacity{resource='cpu'})"
   fetchIntervalSeconds: 10
+  range:
```

Comment:
If I recall correctly v1alpha1 does not have this parameter

Reply:
Oh yes, my bad, it got missed out by me. Will fix this.